src/app/patient-login/patient-login.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
+  <label><h1>Create a new patient</h1></label>
+  <table class="resultsTable" border="1">
+    <thead>
+      <th>New ID</th>
+      <th>First Name</th>
+      <th>Last Name</th>
+      <th>Email</th>
+      <th>Select</th>
+    </thead>
+    <tbody>
+      <tr focus>
+        <td>{{nextId}}</td>
```

Comment:
This should be removed since we don't know the new patient id until a new patient is created.